src/CoreFx.Private.TestUtilities/src/CoreFx.Private.TestUtilities.csproj

```diff
@@ -29,6 +29,9 @@
     <Compile Include="$(CommonPath)\System\PasteArguments.cs">
       <Link>Common\System\PasteArguments.cs</Link>
     </Compile>
+    <Compile Condition="'$(TargetGroup)' == 'uap'" Include="$(CommonTestPath)\System\Threading\ThreadTestHelpers.cs">
```

Comment:
> I don't understand one thing, how are test projects (for instance System.Threading\tests) getting access to RemoteExecutorTestBase without referencing the project?

When compiling the test projects we automatically adding references to all assemblies under the runtime folder so the tests don't have to include these manually